src/app/issuer/components/learningpath-edit-form/learningpath-edit-form.component.html

```diff
@@ -168,42 +168,117 @@ <h2 hlmH2 class="tw-font-bold">3. {{ 'LearningPath.chooseBadges' | translate }}<
 								<!-- Search  -->
-								<div class="tw-pb-2 md:tw-pb-0">
+								<div class="tw-pb-2 md:tw-pb-0 tw-flex tw-relative tw-items-center">
 									<input
 										type="text"
 										name="forminput"
-										changeOrder
-										class="tw-border-solid tw-border-purple tw-bg-white tw-mr-6"
+										class="!tw-bg-white tw-w-full tw-border-1 tw-border-purple min-[880px]:tw-w-[530px] tw-border-solid tw-rounded-lg tw-p-2"
 										placeholder="{{ 'Badge.searchBadges' | translate }}"
 										[(ngModel)]="searchQuery"
 										[ngModelOptions]="{ standalone: true }"
 										hlmInput
 									/>
+									<ng-icon
+										hlm
+										name="lucideSearch"
+										class="tw-w-8 tw-h-8 tw-absolute tw-top-1/2 tw-right-2 -tw-translate-y-1/2 tw-pointer-events-none tw-text-purple"
+									></ng-icon>
 								</div>
+								<!-- Group By  -->
+								<oeb-select
+									class="md:tw-w-[270px] oeb"
+									[options]="groupOptions"
+									[control]="groupControl"
+									[noTopMargin]="true"
+								></oeb-select>
 							</div>
 
-							<div class="tw-grid tw-grid-cols-badgesLp tw-gap-6 tw-mt-4">
-								@for (badge of badgeResults; track badge; let i = $index) {
-									<bg-badgecard
-										[badgeTitle]="badge.name"
-										[badgeImage]="badge.image"
-										[badgeDescription]="badge.description"
-										[badgeSlug]="badge.slug"
-										[issuerTitle]="badge.issuerName ? badge.issuerName : badge.issuer"
-										[badgeIssueDate]="badge.createdAt"
-										[public]="true"
-										[publicUrl]="badge.url"
-										[issuerSlug]="badge.issuerSlug"
-										[competencies]="badge.extension['extensions:CompetencyExtension']"
-										(ngModelChange)="checkboxChange($event, badge)"
-										(checkboxChange)="checkboxChange($event, badge)"
-										[checked]="badgeChecked(badge)"
-										[showCheckbox]="true"
-									></bg-badgecard>
+							@if (groupBy === '---') {
+								<div class="tw-grid tw-grid-cols-badgesLp tw-gap-6 tw-mt-4">
+									@for (badge of badgeResults; track badge; let i = $index) {
+										<bg-badgecard
+											[badgeTitle]="badge.name"
+											[badgeImage]="badge.image"
+											[badgeDescription]="badge.description"
+											[badgeSlug]="badge.slug"
+											[issuerTitle]="badge.issuerName ? badge.issuerName : badge.issuer"
+											[badgeIssueDate]="badge.createdAt"
+											[public]="true"
+											[publicUrl]="badge.url"
+											[issuerSlug]="badge.issuerSlug"
+											[competencies]="badge.extension['extensions:CompetencyExtension']"
+											(ngModelChange)="checkboxChange($event, badge)"
+											(checkboxChange)="checkboxChange($event, badge)"
+											[checked]="badgeChecked(badge)"
+											[showCheckbox]="true"
+										></bg-badgecard>
+									}
+								</div>
+							}
+							@if (groupBy === 'Kategorie' || groupBy === 'Category') {
```

Comment:
This looks very inviting for errors if its actually using the translated values.
However I don't even think thats the case, right?
`groupBy` seems to be using the values of the select, which can be "---", "Category" or "Issuer" and not the labels, correct?